nilearn/plotting/displays/_axes.py

```diff
@@ -26,14 +26,18 @@ class BaseAxes(object):
 
     coord : :obj:`float`
         The coordinate along the direction of the cut.
+    radiological : :obj:`bool`, optional
```

Comment:
Now that you have added `radiological` to the standard doc, you can replace this by: `%(radiological)s`

Reply:
done